tests/Unit/Evolution/Systems/GrMhd/ValenciaDivClean/BoundaryConditions/Test_FreeOutflow.cpp

```diff
@@ -27,6 +28,22 @@ SPECTRE_TEST_CASE("Unit.GrMhd.BoundaryConditions.FreeOutflow",
   pypp::SetupLocalPythonEnvironment local_python_env{""};
   MAKE_GENERATOR(gen);
 
+  const auto face_mesh_index = Index<2>{2};
+  DataVector used_for_size{face_mesh_index.product()};
+  std::uniform_real_distribution<> dist(0.5, 1.0);
+
+  const auto spatial_metric =
```

Comment:
We have functions for generating a random metric that's physical

Reply:
Oh good. Where can I find it?